pkg/api/api.proto

```diff
@@ -78,6 +78,7 @@ message CreateReleaseRequest {
   string sourceAuthor = 7;
   string sourceMessage = 8;
   string sourceRepoUrl = 9;
+  string displayVersion = 10;
```

Comment:
this must be optional

Reply:
yes, but I would keep it this way. the usual convention of treating an empty string as absent is fine for us.

Reply:
as long as we don't expose this in a sort of "public api", I don't mind either way.

E.g. once we generate a documentation/swagger/something out of this, it may be relevant to set it to optional. Since we don't have that, we can keep it non-optional.